package.json

```diff
@@ -5,31 +5,21 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^3.4.2",
+    "css-minimizer-webpack-plugin": "^4.2.2",
     "dedent": "^0.7.0",
-    "extract-loader": "^4.0.3",
-    "file-loader": "^5.0.2",
     "html-webpack-plugin": "^4.3.0",
     "imports-loader": "^0.8.0",
-    "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^7.0.3",
-    "optimize-css-assets-webpack-plugin": "^6.0.1",
-    "postcss": "^8.1",
-    "postcss-loader": "^4.0.3",
+    "mini-css-extract-plugin": "^2.7.5",
+    "sass": "^1.59.3",
```

Comment:
Why is this needed?

Reply:
locally I get `Module Error (from ./node_modules/sass-loader/dist/cjs.js): Cannot find module 'sass'`

Reply:
that is why I added it in the first place.  I take it you're not seeing it?

Reply:
more broadly, we need *something* to compile our SCSS to CSS, and we used to use the now-deprecated `node-sass`.  `sass` is the recommended replacement AFAICT.

Reply:
No I'm not seeing it, but I'm realizing I have `sass` globally installed.

Looking at the Sass docs, I think we have to keep `node-sass` in the package.json: remove `sass`, replace with `node-sass`.

Reply:
Read the [implementation section of sass-loader](https://github.com/webpack-contrib/sass-loader#implementation)'s Github README for more about `sass` versus `node-sass`.